sdk-base/src/main/java/com/mapbox/maps/plugin/viewport/ViewportStatusObserver.kt

```diff
@@ -0,0 +1,15 @@
+package com.mapbox.maps.plugin.viewport
+
+/**
+ * Observer that gets notified whenever [ViewportStatus] changes.
+ */
+fun interface ViewportStatusObserver {
+  /**
+   * Called whenever [ViewportStatus] changes.
+   *
+   * @param from The previous [ViewportStatus], null if the previous [ViewportStatus] is IDLE.
```

Comment:
nit : null is not possible here, probably `ViewportStatus.State(null)` should be here instead.

If IDLE state would be reworked after beta probably won't be relevant anymore

Reply:
Resolved in https://github.com/mapbox/mapbox-maps-android/pull/1069